hue/hue/base/hue-dc.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: hue
+spec:
+  selector:
+    deploymentconfig: hue
+  template:
+    metadata:
+      labels:
+        deploymentconfig: hue
+    spec:
+      containers:
+        - name: hue
+          image: hue
+          command: ["build/env/bin/hue", "runcpserver"]
+          imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: 300m
+              memory: 500Mi
+            limits:
+              cpu: 500m
+              memory: 1Gi
+          env:
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: hue-secret
+                  key: s3_access_key_id
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: hue-secret
+                  key: s3_secret_access_key
+          volumeMounts:
+            - name: "hue-ini"
+              mountPath: /hue/desktop/conf/
+            - name: "hive-site-xml"
+              mountPath: /etc/hive/conf/
+          livenessProbe:
+            failureThreshold: 4
+            httpGet:
+              path: /
+              port: 8000
```

Comment:
Should the port 8000 be exposed for the container?

Reply:
Fixed